ddtrace/llmobs/_integrations/openai.py

```diff
@@ -78,15 +78,20 @@ def _set_base_span_tags(self, span: Span, **kwargs) -> None:
                     span.set_tag_str("openai.organization.id", v or "")
                 else:
                     span.set_tag_str("openai.%s" % attr, str(v))
-        span.set_tag_str("openai.request.client", "AzureOpenAI" if self._is_azure_openai(span) else "OpenAI")
+        client = "OpenAI"
+        if self._is_provider(span, "azure"):
+            client = "AzureOpenAI"
+        elif self._is_provider(span, "deepseek"):
+            client = "Deepseek"
+        span.set_tag_str("openai.request.client", client)
 
     @staticmethod
-    def _is_azure_openai(span):
+    def _is_provider(span, provider="openai"):
```

Comment:
```suggestion
    def _is_provider(span, provider):
```

nit: default args in cases like this can lead to sneaky implicit errors, my preference would be to always explicitly pass a provider. Also looks like we don't actually use the method without specifying the arg

Reply:
makes sense!